src/apache/conf/ssl.conf

```diff
@@ -161,3 +161,17 @@ SSLRandomSeed connect file:/dev/urandom 512
         Header always set Strict-Transport-Security "max-age=63072000; includeSubdomains;"
     </IfDefine>
 </VirtualHost>
+
+#########################
+#        Snapweb        #
+#########################
+Listen 4201
+<VirtualHost *:4201>
```

Comment:
Without SSL support, we cannot integrate it inside Nextcloud, so we need to create a virtualhost on our reverse proxy.